app/javascript/components/widgets/weather/hourly-temps.jsx

```diff
@@ -58,29 +58,27 @@ const RainIcon = styled.img`
 const getHourlyWeather = gql`
   fragment HourlyWeather on Weather {
     hourly {
-      data {
-        temperature
-        time
-        precipProbability
-      }
+      temp
+      dt
+      pop
     }
   }
 `;
 
 const HourlyTemps = ({ weather, hours }) => {
   const hourlyWeathers = compose(
     slice(1, hours + 1),
-    pathOr([], ['hourly', 'data']),
+    pathOr([], ['hourly']),
   )(weather);
   return (
     <Wrapper>
-      {hourlyWeathers.map(({ time, temperature, precipProbability }, idx) => (
-        <Item key={time} index={idx}>
-          <Time>{parseHour(time)}</Time>
-          <Temp>{parseInt(temperature, 10)}°</Temp>
+      {hourlyWeathers.map(({ dt, temp, pop }, idx) => (
```

Comment:
I think `dt` and `pop` are too cryptic in the context of weather and we should alias them.